src/Sentry.Unity/SentryUnity.cs

```diff
@@ -32,7 +32,8 @@ public static void Init(SentryUnityOptions unitySentryOptions)
             // Uses the game `version` as Release unless the user defined one via the Options
             if (unitySentryOptions.Release == null)
             {
-                unitySentryOptions.Release = Application.productName + "@" + Application.version;
+                unitySentryOptions.Release = $"{Application.productName}@{Application.version}";

```

Comment:
Just realized something: Can `productName` be null or an empty string? We might want to check for that before appending `@`

Reply:
It can not be null or empty. But it could be just a whitespace. To what should we default in that case? Just the version number without the @?